src/library/starrating.cpp

```diff
@@ -20,14 +20,26 @@
 #include "library/starrating.h"
 #include "util/math.h"
 
+// Magic number? Explain what this factor affects and how
```

Comment:
AFAICT this closely follows the [QT example](https://doc.qt.io/qt-5.6/qtwidgets-itemviews-stardelegate-example.html), which doesn't provide a low-level explanation either, so I'd like to add some explanations.